SgmlTests/Tests.cs

```diff
@@ -444,6 +443,68 @@ public void Test_XPathDocument_NameTable()
             Assert.AreEqual("abcd", xpathNavigator.Evaluate("string(/html/body)"), "Expecting '/html/body' to be evaluated correctly.");
 
         }
-#endif
+
+        /// <summary>This is a unit-test for the <see cref="SgmlReader.TextWhitespace"/> property's logic.</summary>
+        [Test]
+        public void Invalid_TextWhitespace_enum_values_should_be_dropped()
+        {
+            SgmlReader sgmlReader = new SgmlReader()
+            {
+                TextWhitespace = (TextWhitespaceHandling)0xFF
+            };
+
+            Assert.AreEqual(TextWhitespaceHandling.TrimBoth | TextWhitespaceHandling.OnlyLineBreaks, sgmlReader.TextWhitespace, "The " + nameof(sgmlReader.TextWhitespace) + " property should only respect defined flags bits.");
+        }
+
+        /// <summary>https://github.com/lovettchris/SgmlReader/issues/15</summary>
+        [Test]
+        public void DoBradescoTrailingWhitespaceTest()
+        {
+            // 63.test is originally "bradesco.ofx" from github.com/kevencarneiro/OFXSharp, used under the MIT license.
+            string bradescoOfxText = Tests.ReadTestResource(name: "63.test");
+            bradescoOfxText = bradescoOfxText.Substring(startIndex: 134); // skip first 134 chars of OFX header.
+
```

Comment:
This might be more robust to local repo newline differences:
```csharp
            string bradescoOfxText = Tests.ReadTestResource(name: "63.test");
            var pos = bradescoOfxText.IndexOf("<OFX>");
            bradescoOfxText = bradescoOfxText.Substring(startIndex: pos); // skip OFX header.
```